src/App.tsx

```diff
@@ -15,47 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  LENGHT,
+  ALPABETICALLY,
+  NONE,
+}
+
+type OrderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+function orderGoods(goods: string[],
+  { sortType, isReversed }: OrderOptions) {
```

Comment:
fix code style here, Serhii described it in previous tasks

I'd also suggest rename this function to getOrderedGoods